homeassistant/components/maxcube/binary_sensor.py

```diff
@@ -20,36 +22,53 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         add_entities(devices)
 
 
-class MaxCubeShutter(BinarySensorEntity):
-    """Representation of a MAX! Cube Binary Sensor device."""
+class MaxCubeBinarySensorBase(BinarySensorEntity):
+    """Base class for maxcube binary sensors."""
+
+    _attr_entity_category = EntityCategory.DIAGNOSTIC
```

Comment:
Can you please add a test for the entity category attribute (like the device_class test)?
(Needs adding to both the MaxCubeShutter and the MaxCubeBattery)

Reply:
https://github.com/home-assistant/core/pull/58699/commits/2c8adc3fa697bed7e21cfc3421a20abf4896be19